src/Checkout/CheckoutFormRequestHandlerExtension.php

```diff
@@ -32,14 +31,14 @@ class CheckoutFormRequestHandlerExtension extends Extension
      */
     public function __construct()
     {
-        $this->identityStore = Injector::inst()->get(IdentityStore::class);
+        $this->identityStore = singleton(IdentityStore::class);
```

Comment:
@hchokshi why the departure from Injector in this case?

Reply:
@stevie-mayhew I find `singleton()`/`Injectable::singleton()` a bit easier to read in terms of intent. Same thing at runtime but semantically more clear in my opinion.